plugins/_chatactions.py

```diff
@@ -28,6 +35,38 @@
 from . import LOG_CHANNEL, LOGS, asst, get_string, types, udB, ultroid_bot
 from ._inline import something
 
+# ------------------------- UFoP Bans -------------------------#
+
+
+class UFoPBan:
+    def __init__(self, api_key: str = None):
+        self.api_key = api_key
+
+    def _make_request(
+        self, method: str, url: str, params: dict = None, json_data: dict = None
+    ):
+        headers = {"accept": "application/json", "api-key": self.api_key}
+        try:
+            response = requests.request(
+                method, url, headers=headers, params=params, json=json_data
+            )
+            return response.json()
+        except requests.RequestException:
+            pass
+
+    def get_ufop_ban(
+        self, user_id: int = None, banlist: bool = False
+    ) -> Union[dict, str]:
+        if banlist:
+            url = "https://ufoptg-ufop-api.hf.space/UFoP/bans"
+            payload = {"user_id": user_id}
+            return self._make_request("GET", url, params=payload)
+        else:
+            raise ValueError("Error: banlist must be True")
+
+
+# ------------------------- Huge Thanks to @xtdevs -------------------------#
```

Comment:
you do get a commit attibute, why add "thanks to myself" everywhere?

Reply:
Ahh because slowly over time I used Ultroid as a base to edit it. And if I ever make my repo public then yeh thought it was right thing to do